ember-cli-build.js

```diff
@@ -112,7 +113,7 @@ module.exports = function (defaults) {
                 plugins: postcssPlugins()
             }
         },
-        fingerprint: disabled,
+        fingerprint: {enabled: true},
```

Comment:
Isn't fingerprinting enabled by default? (meaning this line would no longer be necessary)